pdg/src/graph.rs

```diff
@@ -119,6 +119,8 @@ pub struct Node {
     pub kind: NodeKind,
     /// The `Node` that produced the input to this operation.
     pub source: Option<NodeId>,
+
+    pub debug: String,
```

Comment:
Maybe call this `debug_info` instead.

Reply:
Fixed in https://github.com/immunant/c2rust/pull/566/commits/a9fd15ff6a20f7e05758ed8aa83743b9ba6bfb77.